conf/app.ini

```diff
@@ -257,7 +257,7 @@ MODE = console
 ; Buffer length of channel, keep it as it is if you don't know what it is.
 BUFFER_LEN = 10000
 ; Either "Trace", "Debug", "Info", "Warn", "Error", "Critical", default is "Trace"
-LEVEL = Trace
+LEVEL = Info
```

Comment:
I was thinking that was a better default log level. But at this time I wasn't knowing I will push on develop branch, where Trace is indeed a relevant log level.  Do you want me to delete this change ?